client/asset/btc/btc.go

```diff
@@ -460,12 +480,21 @@ func BTCCloneWallet(cfg *BTCCloneCFG) (*ExchangeWallet, error) {
 // newWallet creates the ExchangeWallet and starts the block monitor.
 func newWallet(cfg *BTCCloneCFG, btcCfg *dexbtc.Config, node rpcClient) *ExchangeWallet {
 	// If set in the user config, the fallback fee will be in conventional units
-	// per kB, e.g. BTC/kB. Translate that to sats/B.
+	// per kB, e.g. BTC/kB. Translate that to sats/byte.
 	fallbackFeesPerByte := toSatoshi(btcCfg.FallbackFeeRate / 1000)
 	if fallbackFeesPerByte == 0 {
 		fallbackFeesPerByte = cfg.DefaultFallbackFee
 	}
-	cfg.Logger.Tracef("Fallback fees set at %d %s/vbyte", fallbackFeesPerByte, cfg.WalletInfo.Units)
+	cfg.Logger.Tracef("Fallback fees set at %d %s/vbyte",
+		fallbackFeesPerByte, cfg.WalletInfo.Units)
+
+	// If set in the user config, the fee rate limit will be in units of BTC/KB.
+	// Convert to sats/byte.
+	feesLimitPerByte := toSatoshi(btcCfg.FeeRateLimit / 1000)
```

Comment:
Ah. It is expected to be in BTC/kB. Just need to update the `configOpts` description then.